### DISCUSSION TITLE
Discard Unsaved Changes after editing the page properties

### DISCUSSION THREAD

**Discussion by appecata** [requarks/wiki#3655] [Error / Bug Report]
**Describe the bug**
If you edit page properties (i.e. adding new tags or description) without changing the page content, then save and close.
The pop-up asks "Discard Unsaved changes?" that is strange, because I've saved changes to page properties.

**To Reproduce**
1. Go to any page, select **Edit Page**.
2. In the Edit mode, select **Page**.
3. In the **Page Properties** dialog, add a new tag or change description, and select **OK**.
4. Select **Save** and then **Close**.
The "Discard Unsaved Changes?" pop-up appears.

**Expected behavior**
I would expect the page to be simply closed as I haven't made changes to the page content, and saved new page properties.

**Host Info (please complete the following information):**
 - OS: Docker Container (Linux)
 - Wiki.js version: 2.2.51
 - Database engine:

**Comment by ghost** [comment]
This happens for me as well, randomly. Not all the time.
